### PR TITLE
Updates commerce events to log page events for non-purchases

### PR DESCRIPTION
**Pull request by briancaw**

### CODE REVIEW

File: AppboyKit.js [hunk 1]
```diff
@@ -101,30 +115,35 @@
...
+                              logAppboyEvent(listOfPageEvents[i]);
+                            } catch (err) {
+                              console.log("Error logging page event" + err.messasge);
+                            }
+                        }
+                    }
+                } else if (event.EventDataType == MessageType.PageEvent) {
+                    logAppboyEvent(event);
+                }
                 /** There is no current mapping for the ProductAddToCart, ProductAddToWishlist, ProductCheckout, ProductCheckoutOption, ProductClick,
```
Comment (samdozor) [inline AppboyKit.js]: Minor but can remove this content now for future-readers of the code - all of those commerce event types will now be supported via page events/expansion

Comment (briancaw) [inline AppboyKit.js]: Updated

**Comment by tbreffni** [comment]
@briancaw I don't see any new unit tests covering the new functionality you added. Do you mind adding one or two? I also think a test is specifically needed that checks that Purchase events are handled differently from other commerce events.

**Comment by tbreffni** [comment]
Looks good.

**Pull request merged**